test/black-box/itemimage/ItemImageAddCommand.test.ts

```diff
@@ -116,7 +116,7 @@ describe('ItemImageAddCommand', () => {
         res.expectJson();
         res.expectStatusCode(404);
         expect(res.error.error.success).toBe(false);
-        expect(res.error.error.message).toBe('Invalid image protocol.');
+        expect(res.error.error.message).toBe('Invalid protocol.');
```

Comment:
use new Exception().getMessage()

Reply:
also its better not to test for res.error.error.success, you'll reach the reason for error faster.